js/components/discoveryPanel.js

```diff
@@ -28,9 +26,39 @@ var DiscoveryPanel = React.createClass({
         })
       }),
       icons: React.PropTypes.objectOf(React.PropTypes.object)
+    }),
```

Comment:
I need to drill in and explicitly define proptype for sendDiscoveryClickEvent.

The controller proptype is less important.